test/functions/vass/proc-iter/error-return-expr-in-iter-1.good

```diff
@@ -1,2 +1,2 @@
-error-return-expr-in-iter-1.chpl:1: In iterator 'i11f':
-error-return-expr-in-iter-1.chpl:3: error: returning a value in an iterator
+error-return-expr-in-iter-1.chpl:3: error: 'return' statements with values are not allowed in iterators
+error-return-expr-in-iter-1.chpl:1: note: inside 'i11f', which is declared as an iterator here
```

Comment:
I personally prefer the "In iterator 'foo'" format - unless we've been switching to writing "in X" after the error elsewhere?